File: src/containers/Desafio#1.py
lstPalabra=[]
lstPalabra2=[]
palabra1=str(input("Ingrese la primer palabra"))
palabra2=str(input("Ingrese la segunda palabra"))

if palabra1==palabra2:
    print("Las palabras son Iguales")
else:
    for letra in palabra1:
        lstPalabra.append(letra.lower()) #Para que tome las Mayusculas Tambien
    for letra2 in palabra2:
        lstPalabra2.append(letra2.lower())

    lstPalabra.sort(reverse=False)  #Reverse=False ·· Ordena las Palabras en Orden Alfabetico Ascendente
    lstPalabra2.sort(reverse=False)

    if lstPalabra==lstPalabra2:
        print("Es un anagrama")
    else:
        print("No es un anagrama")

File: src/containers/Desafio#1.js
const prompt=require("prompt-sync")({sigint:true});


var palabra1=prompt("Ingrese la primer palabra");
var palabra2=prompt("Ingrese la segunda palabra");

var arr=palabra1.toLowerCase().split("").sort().join("");
var arr2=palabra2.toLowerCase().split("").sort().join("");


if(palabra1==palabra2)
{
    console.log("Las palabras son iguales");
}
else if(palabra1.length!=palabra2.length)
{
    console.log(false)
}
else
{
    var verdadero=0
    var falso=0

    for(var i=0; i<arr2.length;i++)
    {
        if(arr[i]==arr2[i])
        {
            verdadero++;
        }
        else
        {
            falso++;
        }
    }
 

    if(verdadero>falso)
    {
        console.log(true)
    }
    else
    {
        console.log(false)
    }

}